awscli/customizations/codeartifact/login.py

```diff
@@ -314,6 +348,17 @@ class CodeArtifactLogin(BasicCommand):
                          'domain',
             'required': False,
         },
+        {
+            'name': 'scope',
+            'help_text': 'Associates a scope with your repository for npm '
+                         '(alias for namespace)',
+            'required': False,
+        },
```

Comment:
I'm wondering if we need both of the `--scope` and `--namespace` parameters? And instead we just expose the `--namespace` parameter and build up the documentation around how it applies to npm sort of like how the CodeArtifact docs describe namespaces: https://docs.aws.amazon.com/codeartifact/latest/ug/codeartifact-concepts.html#welcome-concepts-package-namespace.

The main reason that I'm asking/suggesting it is because it is a bit confusing that there are two parameters that are essentially aliases but one of them would only ever apply to a particular tool. Furthermore, in the future we can always add the `--scope` alias (or expose it in a different way that javascript developers would expect) back if `--namespace` is too general so that gives us a bit more flexibility.

Thoughts? @ryansonshine @danielgmyers

Reply:
@kyleknap I think going with the single `--namespace` param is a good idea. I'll put in the change tomorrow

Reply:
@kyleknap all set